data/web/main/management/commands/populatedb.py

```diff
@@ -0,0 +1,133 @@
+import requests
+import json
+import timeit
+from django.core.management.base import BaseCommand
+from django.db import IntegrityError
+from main.models import Rezept, Zutat, Rezept_Zutat
+
+
+class Command(BaseCommand):
+    help = "Fetch script to populate database with ingredients and cocktails."
+
+    def handle(self, *args, **options) -> None:
+
+        start: float = timeit.default_timer()
+
+        # Clean the database
+        self.stdout.write("CLEANING DATABASE FROM ALL ENTRIES..")
+        Rezept_Zutat.objects.all().delete()
+        Rezept.objects.all().delete()
+        Zutat.objects.all().delete()
+
+        url = requests.get(
+            "https://www.thecocktaildb.com/api/json/v1/1/list.php?i=list"
+        )  # api ingredients
+
+        s: str = url.text
+        data = json.loads(s)
+        ingrs = data["drinks"]
+
+        self.stdout.write("STARTING POPULATE TABLE ZUTAT\n")
+
+        # iterate over every ingredient in list and transact to db
+        for ingr in ingrs:
+            if (
+                ingr["strIngredient1"] is None
+                or Zutat.objects.filter(name=ingr["strIngredient1"]).values()
+                is not None
+            ):
+                pass
+            else:
+                try:
+                    z = Zutat(name=ingr["strIngredient1"])
+                    z.save()
+                    self.stdout.write(
+                        f"INSERTING ENTRY {z.name} INTO TABLE 'ZUTAT'.. SUCCESS!"
+                    )
+                except IntegrityError:
+                    self.stderr.write(f"ENTRY {z.name} ALREADY EXISTS.. SKIPPING")
+
+        self.stdout.write("POPULATING TABLE 'ZUTAT' FINISHED!\n\n")
+
+        # fetch recipes
+        for ch in range(97, 123):
+            self.populate_recipe(ch)
+
+        stop: float = timeit.default_timer()
+        exec: float = stop - start
+        self.stdout.write(f"DATABASE TRANSACTIONS COMPLETED IN {round(exec)}s!")
+
+    # helper for REZEPT and REZEPT_ZUTAT, taking an integer as argument that translates to a ascii letter
+    def populate_recipe(self, ch: int):
+        url = requests.get(
+            "https://www.thecocktaildb.com/api/json/v1/1/search.php?s=" + chr(ch)
```

Comment:
Das Skript gibt noch viele Fehlermeldungen aus, weil versucht wird Rezepte doppelt hinzufügen. 's' scheint alle Rezepte zu suchen, die überhaupt den Buchstaben enthalten. Mit 'f' sollten nur Rezepte geladen werden die mit dem Buchstaben anfangen.

```suggestion
            "https://www.thecocktaildb.com/api/json/v1/1/search.php?f=" + chr(ch)
```